### PR TITLE
Add Photopea case study

### PR DESCRIPTION
**Pull request by tomayac**
Changes proposed in this pull request:

- Add the Photopea case study.
- The review happened in the associated [Google doc](https://docs.google.com/document/d/18weBxK38rQRB3dqca-f9DCRl_EBF80EUshOSZoPbic0/edit).

### CODE REVIEW

**Comment by tomayac** [comment]
FYI: @Photopea.

**Review by rachelandrew — APPROVED** [review]

File: site/en/blog/how-photopea-uses-the-file-handling-api/index.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+date: 2023-01-31
+# updated: 2023-01-31
+hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/IRV9bhPXJnRUXrFWfYE5.png
+alt: Photopea logo pattern
+authors:
+  - thomassteiner
+  - ivankutskir
+tags:
+  - capabilities
+  - case-study
```
Comment (tomayac) [inline site/en/blog/how-photopea-uses-the-file-handling-api/index.md:15]:
```suggestion
  - case-study
  - fugu-case-study
```

File: site/en/blog/how-photopea-uses-the-file-handling-api/index.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    var O = W.files;
+    console.log(O);
+    for (var Y = 0; Y < O.length; Y++) {
+      var T = O[Y];
+      T.getFile().then(function(G) {
+        $.YO([G], null, null, null, [T])
+      })
+    }
+  })
+}
```
Comment (tomayac) [inline site/en/blog/how-photopea-uses-the-file-handling-api/index.md:103]:
```suggestion
var N = window.launchQueue;
if (N) {
  var $ = this.UA;
  N.setConsumer(function (W) {
    var O = W.files;
    console.log(O);
    for (var Y = 0; Y < O.length; Y++) {
      var T = O[Y];
      T.getFile().then(function (G) {
        $.YO([G], null, null, null, [T]);
      });
    }
  });
}
```

File: site/en/blog/how-photopea-uses-the-file-handling-api/index.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+authors:
+  - thomassteiner
+  - ivankutskir
+tags:
+  - capabilities
+  - case-study
+  - fugu-case-study
+---
+
+[Photopea](https://www.photopea.com/) is a free online image editor developed by [Ivan Kutskir](https://blog.ivank.net/). Ivan started working on the app in 2012, and maintains a [blog](https://blog.photopea.com/) sharing the major features he adds to Photopea. Photopea can work with PSD (Adobe Photoshop), XCF (GIMP), Sketch (Sketch App), XD (Adobe XD), and CDR (CorelDRAW) formats.
```
Comment (tomayac) [inline site/en/blog/how-photopea-uses-the-file-handling-api/index.md:19]:
```suggestion
## Introduction

(This article is also available in form of a video.)

{% YouTube id="Y40vMQap9fs", startTime="269" %}

[Photopea](https://www.photopea.com/) is a free online image editor developed by [Ivan Kutskir](https://blog.ivank.net/). Ivan started working on the app in 2012, and maintains a [blog](https://blog.photopea.com/) sharing the major features he adds to Photopea. Photopea can work with PSD (Adobe Photoshop), XCF (GIMP), Sketch (Sketch App), XD (Adobe XD), and CDR (CorelDRAW) formats.
```

**Pull request merged**